<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>PlayerDialog</class>
 <widget class="QDialog" name="PlayerDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Player Database</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>6</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QSplitter" name="splitter2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layout6" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel1" >
           <property name="text" >
            <string>Player name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="playerEdit" />
         </item>
        </layout>
       </item>
       <item>
        <widget class="Q3ListView" name="playerList" >
         <property name="minimumSize" >
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="allColumnsShowFocus" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Born</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Died</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Title</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Country</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layout3" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel1_2" >
         <property name="text" >
          <string>Information:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Q3TextBrowser" name="playerView" >
         <property name="minimumSize" >
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Q3TextBrowser</class>
   <extends>Q3TextEdit</extends>
   <header>Qt3Support/Q3TextBrowser</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
